go/billing/api.py

```diff
@@ -228,34 +244,49 @@ def create_transaction_interaction(self, cursor, account_number,
         session_credits = MessageCost.calculate_session_credit_cost(
             session_cost, markup_percent)
 
+        session_unit_credits = MessageCost.calculate_session_unit_credit_cost(
+            session_unit_cost, markup_percent)
```

Comment:
From discussion in RL, plan is to remove session unit credit cost field: it is the only field in the transaction table which isn't relevant to how costs are calculated, so it doesn't really make sense to have it around. Will be easy enough to add later if we need it.